tools/run-tests.py

```diff
@@ -329,6 +337,7 @@ def run_jerry_debugger_tests(options):
     for job in DEBUGGER_TEST_OPTIONS:
         ret_build, build_dir_path = create_binary(job, options)
         if ret_build:
+            print ("\n%sBuild failed%s\n" % (TERM_RED, TERM_NORMAL))
```

Comment:
If I remember correctly there should be no space before `(` character here.

Reply:
Good point, fixed.